File: benchmark/cases/object_create/object_create.js
class Point {
    constructor(x, y, label) {
        this.x = x;
        this.y = y;
        this.label = label;
    }
}

let sum = 0;
for (let i = 0; i < 1000000; i++) {
    const p = new Point(i, i, "item");
    sum += p.x;
}
console.log(sum);

File: benchmark/cases/object_create/object_create.rs
struct Point {
    x: i64,
    y: i64,
    label: &'static str,
}

fn main() {
    let mut sum: i64 = 0;
    for i in 0..1_000_000i64 {
        let p = Point { x: i, y: i, label: "item" };
        sum += p.x;
    }
    println!("{}", sum);
}
